pySDC/projects/DAE/sweepers/semiImplicitDAE.py

```diff
@@ -26,7 +26,7 @@ class SemiImplicitDAE(FullyImplicitDAE):
         0 = g(\vec{U}_0 + \Delta t (\mathbf{Q} \otimes \mathbf{I}_{n_d}) \vec{U}, \vec{z}, \tau),
 
     where
-    
+
```

Comment:
Why was this not fixed by black? Do black and ruff have different rules?

Reply:
No, but black ignores the docstring. Ruff does not